pydis_site/static/css/home/index.css

```diff
@@ -85,4 +85,3 @@ span.repo-language-dot.javascript {
         max-width: none;
     }
 }
-
```

Comment:
Another special rule for css files? 

I guess there must be files then that should not have newlines to the end of the file.